src/components/Basic/Slider/SliderGrid.module.css

```
@@ -23,4 +23,11 @@
 .Grid-Pol {
     width: 100%;
     height: 100%;
+}
+
+.Grid-Pol-Value {
+    width: 2.2vmax;
```

```suggestion
    width: 4vmax;
```
width `2.2vmax` the word 'Medium' wraps but we use it a lot in the examples in the docs so imo we should increase the initial width a bit so at least the provided examples don't look broken out of the box.